### ISSUE TITLE
Not all documented octicon icon names are supported

### ISSUE THREAD

**Issue by ApolonTorq** [retypeapp/retype#70]
The [icon component documentation](https://retype.com/components/icon/) refers to an [octicon search page](https://octicons-primer.vercel.app/octicons/) which includes icon names which `retypeapp` does not support.

For example, the [person-add](https://octicons-primer.vercel.app/octicons/person-add-16) icon results in the following warning:

```
WARNING: [file.md:line] Unexpected Octicon icon "person-add".
```

It is preferable that `retypeapp` supports the icon names defined in the latest version of the octicon content (or at least the latest version at the time of the `retypeapp` release).

**Comment by geoffreymcgill** [comment]
Looks like the `person-add` icon was added in Octicons v13.0.0 and Retype is currently bundling v12.1.0.

We will update Retype to use the latest Octicons release.

**Comment by ApolonTorq** [comment]
Maybe have the retypeapp error message include a link to the octicon release version it was built with?
It makes it clear when its a user typo and when its not. Just thinking out loud here, so feel free to ignore this suggestion.

**Comment by geoffreymcgill** [comment]
Excellent idea. We will include the currently supported version number in the error message.

**Comment by axonasif** [comment]
I had faced this too over several octicons, so my guess was right.

**Comment by geoffreymcgill** [comment]
> I had faced this too over several octicons, so my guess was right.

yes, me too. We are going to improve our process for keeping the Octicons up to date with the latest in each new Retype release.

**Comment by geoffreymcgill** [comment]
We have an automated process now in place to regularly check for Octicon releases.

The Octicons within Retype have been updated and will be available in the upcoming `v1.8.0` release.

**Comment by geoffreymcgill** [comment]
Here's the list of new Octicons that will be included in the `v1.8.0` release.

![new-octicons](https://user-images.githubusercontent.com/62210/126693742-034f3ba8-5fe3-4d49-8fdf-6242e4a2f221.png)

And if you want to reproduce in your own `.md` page, you can use the following table:

```
## New Octicons

Shortcode                 | Octicon                         | Size
---                       | ---                             | ---
`blocked`                 | :icon-blocked:                  | 24
`browser`                 | :icon-browser:                  | 24
`codescan`                | :icon-codescan:                 | 24
`codescan-checkmark`      | :icon-codescan-checkmark:       | 24
`codespaces`              | :icon-codespaces:               | 24
`columns`                 | :icon-columns:                  | 24
`dependabot`              | :icon-dependabot                | 24:
`device-camera`           | :icon-device-camera:            | 16 (scaled up to 24)
`diamond`                 | :icon-diamond:                  | 24
`diff-added`              | :icon-diff-added:               | 16 (scaled up to 24)
`diff-ignored`            | :icon-diff-ignored:             | 16 (scaled up to 24)
`diff-modified`           | :icon-diff-modified:            | 16 (scaled up to 24)
`diff-removed`            | :icon-diff-removed:             | 16 (scaled up to 24)
`diff-renamed`            | :icon-diff-renamed:             | 16 (scaled up to 24)
`duplicate`               | :icon-duplicate:                | 24
`ellipsis`                | :icon-ellipsis:                 | 16 (scaled up to 24)
`file-badge`              | :icon-file-badge:               | 16 (scaled up to 24)
`git-pull-request-closed` | :icon-git-pull-request-closed:  | 24
`git-pull-request-draft`  | :icon-git-pull-request-draft:   | 24
`hash`                    | :icon-hash:                     | 24
`issue-closed`            | :icon-issue-closed:             | 24
`issue-draft`             | :icon-issue-draft:              | 24
`issue-opened`            | :icon-issue-opened:             | 24
`issue-reopened`          | :icon-issue-reopened:           | 24
`key-asterisk`            | :icon-key-asterisk:             | 16 (scaled up to 24)
`logo-gist`               | :icon-logo-gist:                | 16 (scaled up to 24)
`logo-github`             | :icon-logo-github:              | 16 (scaled up to 24)
`mark-github`             | :icon-mark-github:              | 16 (scaled up to 24)
`markdown`                | :icon-markdown:                 | 16 (scaled up to 24)
`meter`                   | :icon-meter:                    | 16 (scaled up to 24)
`paintbrush`              | :icon-paintbrush:               | 16 (scaled up to 24)
`person-add`              | :icon-person-add:               | 24
`repo-clone`              | :icon-repo-clone:               | 16 (scaled up to 24)
`repo-forked`             | :icon-repo-forked:              | 24
`repo-pull`               | :icon-repo-pull:                | 16 (scaled up to 24)
`rows`                    | :icon-rows:                     | 24
`select-single`           | :icon-select-single:            | 16 (scaled up to 24)
`sidebar-collapse`        | :icon-sidebar-collapse:         | 24
`sidebar-expand`          | :icon-sidebar-expand:           | 24
`single-select`           | :icon-single-select:            | 24
`sort-asc`                | :icon-sort-asc:                 | 24
`sort-desc`               | :icon-sort-desc:                | 24
`table`                   | :icon-table:                    | 24
`three-bars`              | :icon-three-bars:               | 16 (scaled up to 24)
```
